Makefile

```diff
@@ -26,12 +26,37 @@ test:
 benchmark:
 	go test -bench . $(shell go list ./... | grep -v /vendor/)
 
+build: BUILD_TAGS ?= ""
 build:
-	go build -v ${LDFLAGS} -o $(PREFIX)bin/test-reporter$(BINARY_SUFFIX)
+	go build -v ${LDFLAGS} -tags ${BUILD_TAGS} -o $(PREFIX)bin/test-reporter$(BINARY_SUFFIX)
+
 
 build-all:
```

Comment:
I question the usefulness of keeping this rule in its current form: it's declaring it will build both linux & OS X binaries, but neither of the rules being executed involves any cross-compilation trickery anymore, so it's not actually possible for a single rule to compile both Linux & Max AFAICT.